### ISSUE TITLE
asyncComputed return only default value

### ISSUE THREAD

**Issue by VadimGuslistov** [foxbenjaminfox/vue-async-computed#93]
Please explain to me why when I run this code I get the default value:
`Vue.use(AsyncComputed);
export default {
  asyncComputed: {
    a: {
      get() {
        return 1;
      },
      default() {
        return 0;
      }
   },
  mounted() {
    console.log(this.a)
  }
}
`

I apologize for the unformed code. For some reason, the code insertion does not work.

**Comment by VadimGuslistov** [comment]
And generally nothing is returned by 'return' from asyncComputed values

**Comment by foxbenjaminfox** [comment]
Your `get` function for `a` isn't `async`, nor does it return a `Promise`. `vue-async-computed` is a library for working with `Promise`s, but nevertheless it does try to "do the right thing" when you return something that is not a `Promise`. To do that, if it finds that you're returning something that isn't a `Promise`, it turns it into one by wrapping it with `Promise.resolve`.

This means that `asyncComputed` values are _always_ computed asynchronously (as the name implies.) That means that they all start out as `null`, and only take on their true values when the returned promise resolves. In your case, the (implicit) `Promise` is immediately resolved, as always with `Promise.resolve`. However, the way `Promise`s work, that won't be visible until the next "tick".

In short, you always need to take into account that `asyncComputed` properties are indeed asynchronous. If what you want is a synchonus computed property, use `Vue`'s built-in `computed` properties. If you're using `asyncComputed`, then you're property will start out as `null`, and only take on its real value when the `Promise` in question resolves, which may be as soon as the next "tick".

**Comment by foxbenjaminfox** [comment]
This seems related to #74, and if so it's likely that #95 will fix it.
